performance/rolftimmermans/turbostreamer/oj.rb

```diff
@@ -3,29 +3,31 @@
 
 __SETUP__
 
-TurboStreamer.encode(encoder: :wankel) do |json|
-  json.article do
-    json.object! do
-      json.author do
-        json.object! do
-          json.extract!($author, :name, :birthyear, :bio)
+TurboStreamer.encode(encoder: :oj) do |json|
```

Comment:
Pass `oj` to both `set_default_encoder` and `encode`?
Can one of them can be omitted?

Reply:
Yea one can be removed